src/content/docs/en/guides/content-collections.mdx

```diff
@@ -631,5 +740,5 @@ const { remarkPluginFrontmatter } = await blogPost.render();
 <p>{blogPost.data.title} — {remarkPluginFrontmatter.readingTime}</p>
 ```
 
-The remark and rehype pipelines only runs when your content is rendered, which explains why `remarkPluginFrontmatter` is only available after you call `render()` on your content entry.  In contrast, `getCollection()` and `getEntryBySlug()` cannot return these values directly because they do not render your content.
+The remark and rehype pipelines only runs when your content is rendered, which explains why `remarkPluginFrontmatter` is only available after you call `render()` on your content entry.  In contrast, `getCollection()` and `getEntry()` cannot return these values directly because they do not render your content.
```

Comment:
Nit: runs -> run so the verb agrees with the subject
```suggestion
The remark and rehype pipelines only run when your content is rendered, which explains why `remarkPluginFrontmatter` is only available after you call `render()` on your content entry. In contrast, `getCollection()` and `getEntry()` cannot return these values directly because they do not render your content.
```